File: jsimple-util/c#/jsimple/util/InvalidFormatException.cs
namespace jsimple.util {

    /// <summary>
    /// This is a generic exception class used to indicate the format of something isn't what it's excepted to be.
    /// 
    /// @author Bret Johnson
    /// @since 5/5/13 4:48 AM
    /// </summary>
    public class InvalidFormatException : BasicException {
        public InvalidFormatException(string message) : base(message) {
        }

        public InvalidFormatException(string message, object arg1) : base(message, arg1) {
        }

        public InvalidFormatException(string message, object arg1, object arg2) : base(message, arg1, arg2) {
        }

        public InvalidFormatException(string message, params object[] args) : base(message, args) {
        }
    }

}

File: jsimple-util/src/main/java/jsimple/util/InvalidFormatException.java
/*
 * Copyright (c) 2012-2014, Microsoft Mobile
 *
 * Permission is hereby granted, free of charge, to any person obtaining a copy
 * of this software and associated documentation files (the "Software"), to deal
 * in the Software without restriction, including without limitation the rights
 * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
 * copies of the Software, and to permit persons to whom the Software is
 * furnished to do so, subject to the following conditions:
 *
 * The above copyright notice and this permission notice shall be included in
 * all copies or substantial portions of the Software.
 *
 * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
 * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
 * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
 * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 * THE SOFTWARE.
 */

package jsimple.util;

/**
 * This is a generic exception class used to indicate the format of something isn't what it's excepted to be.
 *
 * @author Bret Johnson
 * @since 5/5/13 4:48 AM
 */
public class InvalidFormatException extends BasicException {
    public InvalidFormatException(String message) {
        super(message);
    }

    public InvalidFormatException(String message, Object arg1) {
        super(message, arg1);
    }

    public InvalidFormatException(String message, Object arg1, Object arg2) {
        super(message, arg1, arg2);
    }

    public InvalidFormatException(String message, Object... args) {
        super(message, args);
    }
}
